source/material_model/rheology/visco_plastic.cc

```diff
@@ -136,7 +153,7 @@ namespace aspect
             // Choice of activation volume depends on whether there is an adiabatic temperature
             // gradient used when calculating the viscosity. This allows the same activation volume
             // to be used in incompressible and compressible models.
-            const double temperature_for_viscosity = in.temperature[i] + adiabatic_temperature_gradient_for_viscosity*in.pressure[i];
+            const double temperature_for_viscosity = in.temperature[i] + adiabatic_temperature_gradient_for_viscosity*effective_pressure;
```

Comment:
Hmm, I'm not if it is correct to use the effective pressure (i.e., when fluid is present) for this term or other terms below aside from the plasticity. @jdannberg - do the melt material models only use a the solid pressure for the viscous flow laws?